src/CardContent/Infographic/Infographic.tsx

```diff
@@ -10,6 +10,10 @@ const Infographic = ({ media, version }: InfographicProps) => {
   // state to track if the image is visible
   const [isVisible, setIsVisible] = useState(false);
 
+  // states to track the status of the image load, these influence whether to display alt, or MUI Skeleton
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
```

Comment:
these were introduced to accomodate the `Skeleton`. If the element is visible (in viewport), but loading (not yet loaded and not yet errored), then we should show the Skeleton